app/services/signature-service.js

```diff
@@ -125,11 +124,11 @@ export default class SignatureService extends Service {
     return false;
   }
 
-  async removeSignFlow(piece) {
-    const signMarkingActivity = await piece.signMarkingActivity;
-    if (signMarkingActivity) {
-      const signSubcase = await signMarkingActivity.signSubcase;
-      const signFlow = await signSubcase?.signFlow;
+  async removeSignFlow(signFlow) {
+    if (signFlow) {
+      const signSubcase = await signFlow.signSubcase;
+      const signMarkingActivity = await signSubcase.signMarkingActivity;
+      const piece = await signMarkingActivity.piece;
```

Comment:
@ValenberghsSven I seem to remember the order of removing things here was important. Can you double check this please?